### PR TITLE
fix (HowTo): Add more clarification around images for certification docs

### PR DESCRIPTION
**Pull request by hickeyma**
Signed-off-by: Martin Hickey <martin.hickey@ie.ibm.com>

### CODE REVIEW

File: docs/content/howto/helm_chart_redhat_certification.md [hunk 1]
```diff
@@ -69,7 +77,7 @@ The steps for certification are as follows:
 
 ```yaml
 # Build the manager binary
-FROM quay.io/operator-framework/helm-operator:v1.3.2
+FROM quay.io/operator-framework/helm-operator:v1.7.1
```
Comment (spzala) [inline docs/content/howto/helm_chart_redhat_certification.md:79]: Suggestion - here and other similar places, may be good idea to add in comment that use version of your choice or latest version as possible - something like "# Build the manager binary. Use the latest version or version of your choice".

**Review by spzala — COMMENTED** [review]
LGTM except one inline comment. @hickeyma thanks!

File: docs/content/howto/helm_chart_redhat_certification.md (hunk 1 above)
Comment (hickeyma) [reply docs/content/howto/helm_chart_redhat_certification.md:79]: Good point. Updated.

**Comment by hickeyma** [comment]
Thanks for review @spzala. Updated and ready for review again.

**Review by spzala — APPROVED** [review]
lgtm. Thanks for quickly addressing comments @hickeyma

**Pull request merged**